create_release_branch.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -eu
+
+#
+# This script creates release branch and rolls version in trunk.
+#
+
+PROGRAM_NAME=$0
+
+usage() {
+    printf "usage: %s --version=[version]\n""$PROGRAM_NAME"
+    printf "version  Next version which will be updated to poms in trunk after release branch is created\n"
+    exit 1
+}
+
+check_parameters() {
+  if [ -z ${NEW_VERSION+x} ]; then usage; fi
+}
+
+for i in "$@"
+do
+case $i in
+    --version=*)
+    NEW_VERSION="${i#*=}"
+    shift
+    ;;
+    *)
+    ;;
+esac
+done
+
+check_parameters
+
+git fetch --all --tags --prune
+git checkout trunk
+git pull --rebase
+
+CURRENT_VERSION=$(grep -oPm1 "(?<=<version>)[^<]+" pom.xml)
+
+git checkout -b release/$CURRENT_VERSION
+
+while true; do
+    read -rp "Do you want to push new release branch to remote?" yn
```

Comment:
Might be helpful to include the version(s) in this query text. Same thing below.

Reply:
Yep, this is a good one. Added.